x-pack/plugins/uptime/public/components/overview/monitor_list/__tests__/__snapshots__/monitor_list.test.tsx.snap

```diff
@@ -123,80 +123,254 @@ exports[`MonitorList component MonitorListPagination component renders the pagin
+                      "type": "icmp",
                     },
-                    "timestamp": 126,
+                    "timestamp": "126",
                   },
                 ],
-                "summary": Object {
-                  "down": 2,
-                  "up": 1,
-                },
                 "timestamp": "123",
                 "url": Object {},
               },
             },
             Object {
               "monitor_id": "bar",
               "state": Object {
-                "checks": Array [
+                "monitor": Object {},
+                "summary": Object {
+                  "down": 0,
+                  "up": 2,
+                },
+                "summaryPings": Array [
                   Object {
+                    "docId": "bar1",
                     "monitor": Object {
+                      "duration": Object {
+                        "us": 123,
+                      },
+                      "id": "bar",
                       "ip": "127.0.0.1",
-                      "status": "up",
+                      "status": "down",
+                      "type": "icmp",
                     },
-                    "timestamp": 125,
+                    "timestamp": "125",
                   },
                   Object {
+                    "docId": "bar2",
                     "monitor": Object {
-                      "ip": "127.0.0.2",
-                      "status": "up",
+                      "duration": Object {
+                        "us": 123,
+                      },
+                      "id": "bar",
+                      "ip": "127.0.0.1",
+                      "status": "down",
+                      "type": "icmp",
                     },
-                    "timestamp": 126,
+                    "timestamp": "126",
                   },
                 ],
-                "summary": Object {
-                  "down": 0,
-                  "up": 2,
-                },
                 "timestamp": "125",
                 "url": Object {},
               },
             },
           ],
           "totalSummaryCount": 2,
+=======
+        "action": "POP",
+        "block": [Function],
+        "canGo": [Function],
+        "createHref": [Function],
+        "entries": Array [
+          Object {
+            "hash": "",
+            "key": "TestKeyForTesting",
+            "pathname": "/",
+            "search": "",
+            "state": undefined,
+          },
+        ],
+        "go": [Function],
+        "goBack": [Function],
+        "goForward": [Function],
+        "index": 0,
+        "length": 1,
+        "listen": [Function],
+        "location": Object {
+          "hash": "",
+          "key": "TestKeyForTesting",
+          "pathname": "/",
+          "search": "",
+          "state": undefined,
+>>>>>>> avc/metricless
         },
         "loading": false,
       }
     }
+<<<<<<< HEAD
     pageSize={10}
     setPageSize={[MockFunction]}
   />
+=======
+  >
```

Comment:
i wonder why CI is passing with these conflicts in snapshot files.

Reply:
We're skipping these tests now due to https://github.com/elastic/kibana/issues/70386. When we re-enable those tests this should start to fail. I temporarily re-enabled the test so I could fix this snapshot, but obviously this is beyond the scope of this issue.